gensim/test/test_atmodel.py

```diff
@@ -36,9 +34,6 @@
 # increases the bound.
 # Test that models are compatiple across versions, as done in LdaModel.
 
-module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
-datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
-
 # set up vars used in testing ("Deerwester" from the web tutorial)
 texts = [
```

Comment:
This `text` used in many places, maybe move to utils too?

Reply:
hmm, 
1) it's data, not utils/functions
2) maybe in another PR ?

Reply:
1. Yeah, but it's "common resourse", I think that's fine.
2. No need to create distinct PR for this change.

Reply:
1. Ok, I'll fix it.
2. small PR is better to review and faster to merge.